src/events/applicationEvents.ts

```diff
@@ -0,0 +1,144 @@
+// // import { Proposal } from '../models/Proposal';
+// // import { SEP } from '../models/SEP';
+// // import { User, UserRole } from '../models/User';
+import { Event } from './event.enum';
+
+interface GeneralEvent {
+  type: Event;
+  key: string;
+  loggedInUserId: number | null;
+  isRejection: boolean;
+}
```

Comment:
This file is big TODO, to figure out how can we share types, (or will we just copy paste it?)

Reply:
I wonder is there only the types that we need to share, or is there bit more functionality that could be factored out into separate package.

Reply:
Yep, that's the question and had the same idea, with more tighter integration between the two systems we will eventually have to do it, but for now I think it can stay like this.